Kernel/Storage/NVMe/NVMeController.cpp

```diff
@@ -287,7 +287,7 @@ UNMAP_AFTER_INIT ErrorOr<void> NVMeController::create_admin_queue(QueueType queu
         return maybe_error;
     }
     set_admin_queue_ready_flag();
-    m_admin_queue = TRY(NVMeQueue::try_create(0, device_identifier().interrupt_line().value(), qdepth, move(cq_dma_region), cq_dma_pages, move(sq_dma_region), sq_dma_pages, move(doorbell_regs), queue_type));
+    m_admin_queue = TRY(NVMeQueue::try_create(*this, 0, irq, qdepth, move(cq_dma_region), cq_dma_pages, move(sq_dma_region), sq_dma_pages, move(doorbell_regs), queue_type));
```

Comment:
In the previous commit, you changed `irq` to `device_identifier().interrupt_line().value()` and now you're changing it back again. Either change it or don´t change it :^)

Reply:
Lol. I think the `irq` change should have been in the next commit. My bad